toolchain/sem_ir/formatter.cpp

```diff
@@ -217,7 +266,7 @@ auto Formatter::FormatScopeIfUsed(InstNamer::ScopeId scope_id,
 
 auto Formatter::FormatClass(ClassId id) -> void {
   const Class& class_info = sem_ir_->classes().Get(id);
-  if (!ShouldFormatEntity(class_info)) {
+  if (!ShouldFormatEntity(class_info, class_info.body_block_id)) {
```

Comment:
```suggestion
  if (!ShouldFormatEntity(class_info, {class_info.body_block_id})) {
```

Reply:
Instead fixing the code locations with unnecessary curlies. ArrayRef supports construction from a single item.